app/controllers/api/avatars.php

```diff
@@ -48,10 +50,21 @@
 
     $cache = new Cache(new Filesystem(APP_STORAGE_CACHE . '/app-0')); // Limit file number or size
     $data = $cache->load($key, 60 * 60 * 24 * 30 * 3/* 3 months */);
-
     if ($data) {
         //$output = (empty($output)) ? $type : $output;
 
+        $cacheRow = $dbForProject->getDocument('cache', $key);
+        if($cacheRow->isEmpty()){
+            Authorization::skip(fn () => $dbForProject->createDocument('cache',  new Document([
+                '$id' => $key,
+                'dateAccessed' => time(),
```

Comment:
Lets sync with @fogelito once we move away from unix timestamps.

Reply:
Ok

Reply:
Ok